website/source/docs/connect/proxies/envoy.md

```diff
@@ -71,6 +71,7 @@ The dynamic configuration Consul Connect provides to each Envoy instance include
  - Service-discovery results for upstreams to enable each sidecar proxy to load-balance
    outgoing connections.
  - L7 configuration including timeouts and protocol-specific options.
+ - Configuration to [expose specific HTTP paths](/docs/connect/registration/service-registration.html#expose-paths-configuration-reference).
```

Comment:
The next paragraph points to dynamic runtime configuration for more info on these settings, so this might deserve a sentence and a pointer there.

Reply:
Had trouble figuring out how to make it fit, so instead I called out the general config-entries page in that dynamic config paragraph. That way people can go to the default config pages and see these options. Let me know what you think.